environment/integration/helpers.go

```diff
@@ -169,7 +203,7 @@ func initializeDaggerOnce(t *testing.T) {
 	})
 
 	if daggerErr != nil {
-		t.Skipf("Skipping test - Dagger not available: %v", daggerErr)
+		t.Fatalf("Dagger not available: %v", daggerErr)
```

Comment:
🐛 **Bug (Medium):** Changing from Skip to Fatal changes test behavior - tests will now fail instead of being skipped when Dagger is unavailable, which may not be desired for environments where Dagger isn't installed

```suggestion
		t.Skipf("Skipping test - Dagger not available: %v", daggerErr)
```